statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -105,42 +105,70 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	numRows := 8
 	rows := make([][]string, numRows)
 
+	nodeName := wr.presenter.GetNodeName()
+	shardId := wr.presenter.GetShardId()
+	instanceType := wr.presenter.GetNodeType()
+	shardIdStr := fmt.Sprintf("%d", shardId)
+	if shardId == uint64(sharding.MetachainShardId) {
+		shardIdStr = "meta"
+	}
+	wr.instanceInfo.RowStyles[0] = ui.NewStyle(ui.ColorYellow)
+	rows[0] = []string{fmt.Sprintf("Node name: %s (Shard %s - %s)", nodeName, shardIdStr, strings.Title(instanceType))}
+
 	appVersion := wr.presenter.GetAppVersion()
-	rows[0] = []string{fmt.Sprintf("App version: %s", appVersion)}
+	rows[1] = []string{fmt.Sprintf("App version: %s", appVersion)}
 	if strings.Contains(appVersion, core.UnVersionedAppString) {
-		wr.instanceInfo.RowStyles[0] = ui.NewStyle(ui.ColorRed)
+		wr.instanceInfo.RowStyles[1] = ui.NewStyle(ui.ColorRed)
 	} else {
-		wr.instanceInfo.RowStyles[0] = ui.NewStyle(ui.ColorGreen)
+		wr.instanceInfo.RowStyles[1] = ui.NewStyle(ui.ColorGreen)
 	}
 
 	pkTxSign := wr.presenter.GetPublicKeyTxSign()
-	rows[1] = []string{fmt.Sprintf("Public key TxSign: %s", pkTxSign)}
+	rows[2] = []string{fmt.Sprintf("Public key TxSign: %s", pkTxSign)}
 
 	pkBlockSign := wr.presenter.GetPublicKeyBlockSign()
-	rows[2] = []string{fmt.Sprintf("Public key BlockSign: %s", pkBlockSign)}
+	rows[3] = []string{fmt.Sprintf("Public key BlockSign: %s", pkBlockSign)}
+
+	var consensusInfo string
+	countConsensus := wr.presenter.GetCountConsensus()
+	countConsensusAcceptedBlocks := wr.presenter.GetCountConsensusAcceptedBlocks()
 
-	shardId := wr.presenter.GetShardId()
-	shardIdStr := ""
 	if shardId == uint64(sharding.MetachainShardId) {
-		shardIdStr = "meta"
+		consensusInfo = fmt.Sprintf("Count consensus participant: %d | Signed blocks headers: %d", countConsensus, countConsensusAcceptedBlocks)
+
 	} else {
-		shardIdStr = fmt.Sprintf("%d", shardId)
+		consensusInfo = fmt.Sprintf("Consensus accepted / signed blocks: %d / %d", countConsensusAcceptedBlocks, countConsensus)
 	}
-	rows[3] = []string{fmt.Sprintf("ShardID: %s", shardIdStr)}
 
-	instanceType := wr.presenter.GetNodeType()
-	rows[4] = []string{fmt.Sprintf("Instance type: %s", instanceType)}
-
-	countConsensus := wr.presenter.GetCountConsensus()
-	rows[5] = []string{fmt.Sprintf("Consensus group participation count: %d", countConsensus)}
+	rows[4] = []string{consensusInfo}
 
 	countLeader := wr.presenter.GetCountLeader()
-	rows[6] = []string{fmt.Sprintf("Elected consensus leader count: %d", countLeader)}
-
 	countAcceptedBlocks := wr.presenter.GetCountAcceptedBlocks()
-	rows[7] = []string{fmt.Sprintf("Consensus proposed & accepted blocks: %d", countAcceptedBlocks)}
+	rows[5] = []string{fmt.Sprintf("Consensus leader accepted / proposed blocks : %d / %d", countAcceptedBlocks, countLeader)}
+
+	switch instanceType {
+	case string(core.NodeTypeValidator):
+		rewardsPerHour := wr.presenter.CalculateRewardsPerHour()
+		rows[6] = []string{fmt.Sprintf("Rewards estimation:  %s TERD/h (without fees)", rewardsPerHour)}
```

Comment:
remove a space between `estimation:` and `%s`

Reply:
Done.